pyproject.toml

```diff
@@ -7,42 +7,34 @@ max-branches=50
 max-nested-blocks=10
 max-statements=200
 
-[tool.isort]
-profile = "black"
-src_paths = ["src", "tests/ci", "tests/sqllogic"]
-
 [tool.pylint.FORMAT]
 #ignore-long-lines = (# )?<?https?://\S+>?$
 
 [tool.pylint.'MESSAGES CONTROL']
           # pytest.mark.parametrize is not callable (not-callable)
 disable = '''
-          pointless-string-statement,
-          line-too-long,
           missing-docstring,
           too-few-public-methods,
           invalid-name,
           too-many-arguments,
-          keyword-arg-before-vararg,
           too-many-locals,
           too-many-instance-attributes,
           cell-var-from-loop,
           fixme,
           too-many-public-methods,
           wildcard-import,
-          unused-wildcard-import,
-          singleton-comparison,
-          not-callable,
           redefined-outer-name,
           broad-except,
           bare-except,
-          no-else-return,
-          global-statement,
-          f-string-without-interpolation,
-          consider-using-with,
-          use-maxsplit-arg,
```

Comment:
this is excessive check for ci scripts

Reply:
Again, this one was abused in the place pylint found it https://github.com/ClickHouse/ClickHouse/pull/70013/files#diff-b3f5f3ea620ee8e310f6bdfab21608735e58c4d04402cd5a70fd267047b36a1aR575-R592

Using `maxsplit` is a good practice to save the memory.